src/components/progressBar/progressBar.style.js

```diff
@@ -0,0 +1,33 @@
+import {color} from '../../theme/style';
```

Comment:
in theme we have index. you can import everything from "../../theme" - that's the idea to have an index.

Reply:
I tried doing this

Reply:
import {color, size} from '../../theme'; but i keep getting error

Reply:
solved the issue, it was the naming of the folder